protected/humhub/modules/space/views/space/about.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+use humhub\modules\space\models\Space;
+use humhub\modules\space\widgets\AboutPageSidebar;
+use humhub\modules\content\widgets\richtext\RichText;
+use humhub\modules\user\widgets\Image;
+
+/**
+ * @var Space $space
+ * @var array $userGroups
+ */
+?>
+
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <?= Yii::t('SpaceModule.base', '<strong>About</strong> this Space') ?>
```

Comment:
I think this would be a perfect moment to use something like this would it not? The use case would better define that it's about the given space rather than just a "Space" @luke- @fae7em? Would also make it so that we'd not have to worry about translation questions regarding the term "Space" being translated into other languages just my opinion :thinking:

`<?= Yii::t('SpaceModule.base', '<strong>About</strong> <strong>{spaceName}</strong>', ['spaceName' => $space->name]) ?>`

Reply:
Also we should not name the menu item `About Space` but rather `About`

Reply:
I'd also like this done here as well https://github.com/humhub/humhub/pull/4442#discussion_r501694563 because I feel that it'd be similar in thinking :thinking:

Reply:
Currently there are no plans to reduce the usage or rename the fixed term "Space" which is a core element of HumHub.

We discussed the wording for a longer time in the team and came to the following result:

- **About** (Menu entry)
- **About the Space** (Title of the About page)
- **About your membership** (Sidebar Snippet title)

If for some reason the wording doesn't work at all, we can of course still change it, but please start a separate issue here.